### PR TITLE
Added some games from Epic Games and GOG

### PR DESCRIPTION
**Pull request by Dei657**

### CODE REVIEW

File: gamedb.yaml [hunk 1]
```diff
@@ -232,6 +232,25 @@
   banner: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/0decbbe8e7f4bbda5ecf7be75866985e.png
   status: playable
 
+- name: LEGO® Batman™: The Videogame
```
Comment (alkazar) [inline gamedb.yaml:235]: Titles with ':' need to be in quotes i.e.
"LEGO Batman: The Videogame"

**Review by alkazar — CHANGES_REQUESTED** [review]
Thank you for taking the time to submit you testing results!

File: gamedb.yaml [hunk 2]
```diff
@@ -232,6 +232,25 @@
...
+  id: Grackle
+  banner: steam:213330
+  status: playable
+  notes:
+    - Displays a Windows dialog during initial setup
+  
+- name: 'LEGO Batman 3: Beyond Gotham'
+  platform: epic-store
+  id: Gull 
+  banner: steam:313690
```
Comment (alkazar) [inline gamedb.yaml:252]: I just noticed there isn't a status for LEGO Batman 3. Is that intentional?

**Comment by Dei657** [comment]
During the videos, the game partially lags.  I have not decided what status to give.

**Pull request merged**